readme.md

```diff
@@ -115,6 +115,15 @@ Run `brew cask install suspicious-package` or [download manually](http://www.mot
 [![](screenshots/SuspiciousPackage.png)](http://www.mothersruin.com/software/SuspiciousPackage/)
 
 
+### [QLVideo](https://github.com/Marginal/QLVideo)
+
+> Preview most types of video files, as well as their thumbnails, cover art and metadata
+
+Run `brew cask install qlvideo` or [download manually](https://github.com/Marginal/QLVideo/releases/latest)
+
+[![](https://raw.githubusercontent.com/Marginal/QLVideo/master/img/preview.jpeg)](https://github.com/Marginal/QLVideo)
```

Comment:
The screenshot needs to be included in this repo like the others and should be the same size as the existing ones and with shadow.